src/routes/tools/wildfire/_ExploreData.svelte

```diff
@@ -97,17 +98,24 @@
   });
 
   $: if (Array.isArray($dataStore) && $dataStore.length) {
-    statsData = $dataStore.filter((d) => d.mark !== "area");
     dataByDate = getDataByDate(flattenData($dataStore));
     noData = Math.max(...$dataStore.map((d) => d.values.length)) === 0;
   } else {
-    statsData = null;
     dataByDate = null;
   }
 
   afterUpdate(() => {
-    showMissingDataMsg = $locationStore.boundaryId !== "locagrid";
-    showNoDataMsg = $locationStore.boundaryId === "locagrid" && noData;
+    if (!activeTab) {
```

Comment:
yikes. the wildfire has just been a painful dataset to work with.

Reply:
Yeah, it's why I've been talking with Brian about at least showing the % of grid cells with no data values in a boundary area.